src/MAX40080/MAX40080.cpp

```diff
@@ -111,13 +111,11 @@ int MAX40080::read_register(uint8_t reg, uint8_t *buf, uint8_t len/*=1*/)
         r_Len += 1; // +1 for PEC byte
     }
 
-    m_i2c->requestFrom((char)m_slave_addr, (char)r_Len, false);
```

Comment:
Please keep third parameter, 
pass true, to explicitly define stop condition.

Reply:
done